sdl2-hyper-sonic-drivers/examples/adl-example.cpp

```diff
@@ -31,19 +31,19 @@ void adl_test(const OplEmulator emu, const OplType type, std::shared_ptr<audio::
         return;
 
     auto adlFile = std::make_shared<ADLFile>(filename);
-    ADLDriver adlDrv(opl, adlFile);
+    ADLDriver adlDrv(opl, audio::mixer::eChannelGroup::Music);
```

Comment:
deprecated. do not pass opl, but a "device" instead.

group set a default one? volume and pan yes